### ISSUE TITLE
Cross-compiling to Windows breaks playing .WebM(VP9) video

### ISSUE THREAD

**Issue by zaevi** [godotengine/godot#50862]
### Godot version

3.3.2.stable

### System information

Windows10 21H1

### Issue description

Webm(VP9) video can't play in official engine and crash it, **but works fine in engine compiled from Windows**.

Considering official Windows build is compiled from Linux, I cross-compiled the engine on Linux (fedora34, in docker desktop), and it also crashes.

VP8 always works fine.

There is something wrong with webm module when Cross-compiling for Windows from Linux.

might related to #37648

### Steps to reproduce

1.Play webm(vp9) video in official build, it will crash.
2.Compile from Windows and play webm(vp9), works fine.

### Minimal reproduction project

I made minimal project contains vp8 and vp9 videos so you can reproduce it easily:
[webm_bug.zip](https://github.com/godotengine/godot/files/6874228/webm_bug.zip)

(I want to upload the engine compiled from Windows, but it‘s too big)

**Comment by Calinou** [comment]
Official binaries are cross-compiled for Windows from Linux too, so the issue must be coming from somewhere else.

**Comment by zaevi** [comment]
> Official Godot binaries are cross-compiled for Windows from Linux too, so the issue must be coming from somewhere else.
> 
> Do you have `yasm` installed in the container? Does it print a warning about YASM not being available while building near the top of the SCons log?

Yes, I've installed `yasm`. But what I mean is the official binary (from Linux) can't play vp9, but the build I compiled from Windows can.

**Comment by zaevi** [comment]
Today I tried to compile the engine WITHOUT `yasm` installed and it can play vp9 videos well. So it should be YASM-related that causes VP9 crash the engine.

**Comment by akien-mga** [comment]
That's correct, I seem to have removed yasm from the base mono container (used for `Dockerfile.windows`) by mistake in https://github.com/godotengine/build-containers/commit/b409e6b9e9841aa5a5765c203ee9b9ebbba12180

So the issue will be fixed in 3.4 then, at the cost of unoptimized WebM decoding (but that's better than a crash for sure).
